debugger/ptraceLib/enclave_context.c

```diff
@@ -590,7 +595,7 @@ int oe_get_enclave_thread_xstate(
     }
 
     if (xstate_size >
-        (ssa_info.frame_byte_size - sizeof(struct user_regs_struct)))
+        (ssa_info.frame_byte_size - (int64_t)sizeof(struct user_regs_struct)))
```

Comment:
There doesn't seem to be any reason why `oe_get_enclave_thread_xstate` is defined to take `xstate_size` as `int64_t` and not an unsigned type. It's dealing strictly with a non-negative size and either being used with `size_t` or `uint64_t` values. It would also avoid an unnecessary cast in debugger/ptraceLib/oe_ptrace.c if it was just defined as `size_t` by default.

Similarly for `oe_set_enclave_thread_xstate` below.

Reply:
Can you also remove the `int64_t` cast here since these are all dealing with unsigned values now?